packages/markdown-slate/package.json

```diff
@@ -76,7 +76,7 @@
     "webpack-cli": "^3.3.5"
   },
   "dependencies": {
-    "@accordproject/markdown-common": "0.3.1",
+    "@accordproject/markdown-cicero": "^0.3.1",
```

Comment:
I think for the lerna packages, we have used exact versions in mutual dependencies (at least in Ergo and Cicero). I would probably keep it that way.

Reply:
Fixed